lighthouse-core/config/perf.json

```diff
@@ -1,114 +1,6 @@
 {
-  "passes": [{
-    "recordNetwork": true,
-    "recordTrace": true,
-    "pauseBeforeTraceEndMs": 5000,
-    "useThrottling": true,
-    "gatherers": [
-      "image-usage",
-      "viewport-dimensions",
-      "dobetterweb/domstats",
-      "dobetterweb/tags-blocking-first-paint",
-      "dobetterweb/optimized-images"
-    ]
+  "extends": "lighthouse:default",
```

Comment:
this is a lie depending on the current check just being truthiness of `extends`, right?